pdf_gui.py

```diff
@@ -70,13 +70,20 @@ def __init__(self, master):
         ### END OF TO-DO. ###
         
         # Canvas creation to display PDF document pages.
-        self.canvas = Canvas(self.master, bg="white")
+        self.canvas = Canvas(self.master, width=854, height=480, bg="white")
```

Comment:
Adding width and height may be unnecessary since we want the canvas' size to not be restricted, but this will stay for now.

Reply:
I will probably also need to change the background color since I realized that the white background makes it hard to see the edges of the page and that the page ends up blending in (a fault on my end). If you are going to change it, I recommend a color like gainsboro or some other off-white color. You can search up the colors that tkinter supports online.